src/three-components/TextureUtils.ts

```diff
@@ -192,9 +184,13 @@ export default class TextureUtils extends EventDispatcher {
         environmentMapLoads = this[$loadGeneratedEnvironmentMap]();
       }
 
-      const [environmentMap, skybox] =
+      let [environmentMap, skybox] =
           await Promise.all([environmentMapLoads, skyboxLoads]);
 
+      if (skybox != null) {
+        skybox = this.gaussianBlur(skybox, 0.004, GammaEncoding);
```

Comment:
Doesn't this return a reference to a new, distinct render target? If so, who manages the lifecycle of this instance? When does it get disposed?

Reply:
Sometimes it is the same renderTarget (if the input is jpg); if it is hdr then gaussianBlur disposes of the one you gave it and returns this new one in its place.

Reply:
Doesn't that mean we're possibly disposing of the cached render target?

Reply:
Let's fix this in a following PR to extend our usage of your cachingPolicy.

Reply:
To be clear, we're no longer disposing of the cached render target, but we are leaking backgrounds.